examples/http/README.md

```diff
@@ -0,0 +1,27 @@
+# HTTP Example
+
+This is a simple example that demonstrates how to use the OpenTelemetry SDK 
+to instrument a simple HTTP based Client/Server application. 
+The example creates the **Root Span** on the client and sends the distributed context
```

Comment:
simply `context` should be enough here (`DistributedContext` is something very specific, and you are also injecting a `SpanContext`, etc).

Reply:
I have addressed this in fa3dc34